lib/pingen/client.rb

```diff
@@ -21,6 +22,14 @@ def documents
       get_request("/document/list")
     end
 
+    def get(id)
+      get_request("/document/get/id/#{id}")
+    end
+
+    def upload(pdf)
```

Comment:
Not sure if we need anything else for the moment, maybe option to automatically also queue it for sending. This is possible with the API, but I'd add it as we go.

Reply:
Yes, fine for me